packages/paste-core/utilities/grid/src/helpers.tsx

```diff
@@ -26,6 +26,21 @@ export const getOuterGutterPull = (theme: ThemeShape, gutter?: Space): Margin =>
   return 'auto';
 };
 
+// Gets the gutter and returns the value to be used as padding for Columns
+export const getColumnGutters = (theme: ThemeShape, gutter?: Space): Padding => {
+  if (Array.isArray(gutter)) {
+    return (gutter as SpaceOptions[]).map((value: SpaceOptions) => {
+      return `${theme.space[value]}` as SpaceOptions;
+    });
+  }
+
+  if (gutter) {
+    return `${theme.space[gutter as SpaceOptions]}` as SpaceOptions;
+  }
```

Comment:
This is identical to the code in `getOuterGutterPull`. It might be worth pulling it out of both into a shared function. Something like `getResponsiveValueFromTheme` or something.

The only difference is the returned default values of `auto` and `space0`.

Reply:
This one also returns the positive value of the token instead of the negative value like `getOuterGutterPull`. Still worth pulling into a shared function?

Reply:
See updated edit